pkg/storage/sds/models.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package sds
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/doc/jose"
+)
+
+// StructuredDocument represents a Structured Document
```

Comment:
StructuredDocument represents a Structured Document

seems like an odd start to a sentence :).